recipes/sequenticon/meta.yaml

```
@@ -0,0 +1,53 @@
+{% set name = "sequenticon" %}
+{% set version = "0.1.6" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/Edinburgh-Genome-Foundry/sequenticon/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 05bdf8da346db16c3c82aa78cffac3f29d71695b322f567d4bc37b9dab976705
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  run_exports: 
+    - {{ pin_subpackage('sequenticon', max_pin="x.x") }}
+
+requirements:
+  host:
+    - python 
+    - setuptools
+    - pip
+  run:
+    - python 
+    - biopython
+    - pydenticon
+    - snapgene-reader
+    - flametree
+    - pdf-reports
+
+test:
+  source_files: 
+    - tests/
+  imports:
+    - {{ name|lower }}
+  requires:
+    - matplotlib
```

If you want to move this forward, temporarily constrain the python in the test:

```suggestion
  ## NB: python constraint is temp workaround; please try removing in future builds
  - python <3.10
  - matplotlib
```